test/memory.test.js

```diff
@@ -565,6 +574,33 @@ describe('Memory connector', function() {
       });
     });
 
+    it('should handle constructor.prototype', function(done) {
```

Comment:
is this related to https://github.com/strongloop/loopback-next/pull/6676/files#diff-cc4e314e58419ab950165ca69afc20c509017bff0440022ab4e1afd5a69a342aR183 ?

Reply:
yes